### PR TITLE
Add Grade Progress feature

### LINKED ISSUE #5: As a user, I can easily input my students data
so that we can include our students in the list

### LINKED ISSUE #24: As a user ready to start using the app, I can include the grade progress of my students
...so that I can track their progress.

### PR DESCRIPTION
**Pull request by AshiqurRah**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/GradeProgressCommand.java [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    public static final String COMMAND_WORD = "gradeProgress";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Adds grade progress to the person identified "
+            + "by the index number used in the last person listing. "
+            + "Existing grade progress will not be modified.\n"
+            + "Parameters: INDEX (must be a positive integer) "
+            + "gp/ [GRADEPROGRESS]\n"
+            + "Example: " + COMMAND_WORD + " 1 "
+            + "gp/ MATH:A";
```
Comment (phyliciachristel) [inline src/main/java/seedu/address/logic/commands/GradeProgressCommand.java]: maybe you can use the const prefix here just for consistency's sake with the other command classes

**Review by phyliciachristel — APPROVED** [review]
other than this, LGTM congrats !!!!!!!!!!!!!!

File: src/main/java/seedu/address/logic/commands/GradeProgressCommand.java (hunk 1 above)
Comment (AshiqurRah) [reply src/main/java/seedu/address/logic/commands/GradeProgressCommand.java]: Noted, thank you

**Comment by AshiqurRah** [comment]
lgtm!
-farisa

**Pull request merged**